feat(plane_master) adds blur filter over open space

sephirothseeker opened the pull request:
Добавляет модный фильтр блюра на открытые пространства, демонстрирующие нижний з-уровень, благодаря чему стало сложней в них упасть, перепутав с нормальным полом.

![openspace_blur](https://user-images.githubusercontent.com/107106680/175721040-ad3941e4-109b-49a4-9541-e045b2d0cf3a.png)

<details>
<summary>Чейнджлог</summary>

```yml
🆑
rscadd: Теперь нижние з-уровни выглядят размытыми
/🆑
```

</details>

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.
- [x] Я ознакомился c [Guide to Contribute](https://github.com/ChaoticOnyx/OnyxBay/blob/dev/docs/contributing.md).

sephirothseeker committed:
feat(plane_master) adds blur filter over open space

Adds /plane_master/openspace_blur which creates blur over open space

TobyThorne commented:
С одной стороны - информативно. С другой - чёт прям совсем уродливый блюр.

Tsurupeta commented:
Уменьши затемнение нижнего плейна и поставь блюр с 1 на 0.5, чтобы выглядело лучше.

sephirothseeker committed:
tweak(plane_master) calibrating openspace blur + darkness over openspace

Slight calibration of blur filter over openspace + increasing openspace darkness alpha

sephirothseeker commented:
Слегка повысил альфу у темноты и ослабил блюр
Выглядит определённо лучше, чем то что стоит сейчас

![openspace_noblur](https://user-images.githubusercontent.com/107106680/175779938-91df87be-3da3-4b0f-a3d7-d7aaadc4b8e5.png)
![openspace_blur4](https://user-images.githubusercontent.com/107106680/175780016-18345a92-b40d-4f46-92f5-0d1325113f74.png)

Artemonim commented:
Выглядит здорово!

Zert0X commented:
Требую скрин шахты лифта в карго

sephirothseeker commented:
![cargo_elevator](https://user-images.githubusercontent.com/107106680/175782894-7efbc733-770e-4a3e-8f3a-dada91952740.png)

Zert0X commented:
вроде без блюра

Tsurupeta commented:
> вроде без блюра

Блюр просто subtle

sephirothseeker commented:
> вроде без блюра

Размывает. Вот с блюром на 1 пиксель
![image](https://user-images.githubusercontent.com/107106680/175787293-a6b45c96-4540-4262-94cc-c217f8e1fa09.png)

sephirothseeker commented:
> вроде без блюра

всё он размывает, вот пример с блюром на 1 пиксель
![cargo_blur](https://user-images.githubusercontent.com/107106680/175787342-afc8bb60-ebff-449e-808a-87df4b836308.png)

sephirothseeker committed:
fix(plane_master) makes mobs appear blurry and darker on lower z-shadows

Places z-shadow mobs on openspace plane so blur filter and over_darkness apply to them.

sephirothseeker committed:
tweak(plane_master) makes over_openspace_darkness transparent to mouse

Makes over_openspace_plane transparent to clicks so you could interact witht z-shadow mobs without using right-click or altclick

kreeperHLC reviewed (approved):
Я хз чей ты мультиакк, но свою работу ты выполняешь на ура
